From 3ea3059f3c86bd2333284bbce85350f778f8dceb Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Thu, 12 Jul 2018 15:46:18 +0800
Subject: [PATCH 201/205] memory: ti-emif-sram: fix forwarding porting changes

emif_instance is introduced only by mainline to manage all these local data.
So we do the related right data references for resume.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/memory/ti-emif-pm.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/memory/ti-emif-pm.c b/drivers/memory/ti-emif-pm.c
index ac14626..3b37a48 100644
--- a/drivers/memory/ti-emif-pm.c
+++ b/drivers/memory/ti-emif-pm.c
@@ -252,7 +252,7 @@ int ti_emif_get_mem_type(void)
 #ifdef CONFIG_PM_SLEEP
 static int ti_emif_resume(struct device *dev)
 {
-	unsigned long tmp =  __raw_readl((void *)ti_emif_sram_virt);
+	unsigned long tmp =  __raw_readl((void *)emif_instance->ti_emif_sram_virt);
 
 	/*
 	 * Check to see if what we are copying is already present in the
@@ -261,7 +261,7 @@ static int ti_emif_resume(struct device *dev)
 	 * the PM code
 	 */
 	if (tmp != ti_emif_sram)
-		ti_emif_push_sram(dev);
+		ti_emif_push_sram(dev, emif_instance);
 
 	return 0;
 }
-- 
1.7.9.5

